BigriverBookstore-apple/Sources/Utils/DateExtensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DateExtensions.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 07/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    enum Format: String {
+        case yyyymmdd = "YYYY-MM-DD"
+        case mmddyyyy = "MM-DD-YYYY"
+    }
+
+    func stringWith(format: Format = .yyyymmdd) -> String {
```

Comment:
@c-Viorel Since DateFormatter() creation is pretty expensive, I propose to use a static one. I'll provide you some examples.